auth-external-api/src/main/java/uk/gov/di/authentication/external/lambda/TokenHandler.java

```diff
@@ -150,12 +158,22 @@ public APIGatewayProxyResponseEvent tokenRequestHandler(APIGatewayProxyRequestEv
 
             authorisationCodeService.updateHasBeenUsed(authCodeStore.getAuthCode(), true);
 
+            String subjectID = authCodeStore.getSubjectID();
+            UserProfile userProfile = dynamoService.getUserProfileFromSubject(subjectID);
+            String internalPairwiseId =
+                    userProfile.getSalt() == null
+                            ? AuditService.UNKNOWN
+                            : ClientSubjectHelper.calculatePairwiseIdentifier(
+                                    subjectID,
+                                    configurationService.getInternalSectorUri(),
+                                    SdkBytes.fromByteBuffer(userProfile.getSalt()).asByteArray());
```

Comment:
if userProfile.getSalt() is null will result in NPE

Reply:
this is fine, it was just the way it looked in GH, fine when looking at the actual code